### ISSUE TITLE
To check invitation letter template

### ISSUE THREAD

**Issue by ekmatts1** [OpusCapita/businessnetwork#14]
![image](https://cloud.githubusercontent.com/assets/28104515/25794975/780cdfec-33d3-11e7-8ada-21b46c5bbfd6.png)
Invitation letter needs to be edited by uploading buyer logo by buyer and to edit  footer text. In addition also to be clickable and enlarged to check result before sneding.

**Comment by gr4per** [comment]
we are missing footer text atm.
please clarify if need to be customer specifc
else we can cover this by full generic template

**Comment by gr4per** [comment]
Should we still consider customizing the footer or do we plan to use customization ?

**Comment by ekmatts1** [comment]
Yes, we should - can we build a template for upload of name, title, phone no, email, domain etc or should we take this data from key-buyer/user?

**Comment by ekmatts1** [comment]
Firstname Lastname

Title Goes Here

OpusCapita

Street Adress

FI-00000 Town, Country

Tel. +358 00 000 0000

Mobile +358 00 000 0000 firstname.lastname@opuscapita.com www.opuscapita.com
